presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestIntegrationSmokeTest.java

```diff
@@ -228,6 +228,16 @@ public void testNonQueryAccessControl()
     }
 
     @Test
+    public void testViewCaseSensitivity()
```

Comment:
This is testing engine functionality, so it should go in `AbstractTestDistributedQueries`. This class is for a limited set of tests that can quickly test the basic functionality of a connector (and thus is fast enough to run for every connector as part of the normal build).

Some existing tests in this class like `testViewAccessControl` are actually in the wrong place and should be moved out.